.github/workflows/builder_nodejs_slsa3.yml

```diff
@@ -91,6 +107,42 @@ jobs:
       # TODO(#888): Publish to GitHub release.
 
       # TODO(#1669): Publish to npm. Note that we need to upload our own bundle to npm.
-      # - name: Publish to npm registry
-      #   run: |
-      #     npm publish # ...
+      # TODO(#1669): Use mainline version of npm
+      - name: Clone npm fork
+        id: clone-fork
+        run: |
+          set -euo pipefail
+
+          # Clone npm fork.
+          cd "$(dirname "$(which node)")"
+          mkdir -p node_modules
+          cd node_modules
+          git clone https://github.com/ianlewis/cli.git npm
```

Comment:
unresolving since I don't see the changes and this is an important one

Reply:
Sorry. I'm in the process of pushing the changes and writing the responses and GitHub doesn't allow you to resolve comments as part of a review.

I pinned the digest and updated it so that it fetches the digest to checkout with --depth 1